examples/dataset.py

```diff
@@ -0,0 +1,125 @@
+from random import randint
+
+from pydomo.datasets import Column
+from pydomo.datasets import DataSetRequest
+from pydomo.datasets import Policy
+from pydomo.datasets import PolicyFilter
+
+def datasets(domo):
```

Comment:
@brandontysonmoss FYI this is saying there should be two blank lines before the `def datasets(domo):` line